polkadot-protocol/protocol-components/transactions/transactions-formats.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Transactions Formats
```

Comment:
Not sure if it was ever discussed, maybe "Extrinsic Formats"

Reply:
I do like "Extrinsic Formats" more, but as the name of the whole section is "Transactions," I'm unsure about this one

Reply:
Yeah, we'd have to rename it

Reply:
@0xLucca @eshaben @dawnkelly09 do you guys agree on this?

Reply:
We can always do this kind of renaming later too, as to not hold up merging of PRs

Reply:
Yes please, let's do that 🙏 

Anyways, will jot this down to raise this conversation in the future. Let's move on with the formatting review now 🚀

Reply:
If are ok with the content, please do approve the PR so we erin and dawn can take a look at it

Reply:
It would make sense to rename it to extrinsics. Lets discuss this later